sdk/python/feast/feature_store.py

```diff
@@ -255,7 +256,9 @@ def apply(
 
     @log_exceptions_and_usage
     def get_historical_features(
-        self, entity_df: Union[pd.DataFrame, str], feature_refs: List[str],
+        self,
```

Comment:
I think `feature_refs` is misleading, what about changing it to `features` and having `features` support either a `List[str]` or `FeatureService` (we'll need to maintain backward compat for one minor version). Also, it's not clear why we need to support `List[FeatureService]`.

Reply:
Yeah, I think changing to `features` makes sense.

Re: List[FeatureService], I think I goofed. Just a single FeatureService is more appropriate.